src/test/java/bio/terra/pdao/bigquery/BigQueryPdaoTest.java

```diff
@@ -56,6 +97,62 @@ public void basicTest() throws Exception {
         Assert.assertThat(exists, is(equalTo(false)));
     }
 
+    @Test
+    public void datasetTest() throws Exception {
+        // Create a random study.
+        StudySummaryModel studySummary = connectedOperations.createTestStudy("ingest-test-study.json");
+
+        // Stage tabular data for ingest.
+        String targetPath = "scratch/file" + UUID.randomUUID().toString() + "/";
+
+        String bucket = dataRepoConfiguration.getIngestbucket();
+
+        BlobInfo participantBlob = BlobInfo
+            .newBuilder(bucket, targetPath + "ingest-test-participant.json")
+            .build();
+        BlobInfo sampleBlob = BlobInfo
+            .newBuilder(bucket, targetPath + "ingest-test-sample.json")
+            .build();
+        BlobInfo fileBlob = BlobInfo
+            .newBuilder(bucket, targetPath + "ingest-test-file.json")
+            .build();
+
+        try {
+            storage.create(participantBlob, readFile("ingest-test-participant.json"));
+            storage.create(sampleBlob, readFile("ingest-test-sample.json"));
+            storage.create(fileBlob, readFile("ingest-test-file.json"));
+
+            // Ingest staged data into the new study.
+            IngestRequestModel ingestRequest = new IngestRequestModel()
+                .format(IngestRequestModel.FormatEnum.JSON);
+
+            connectedOperations.ingestTableSuccess(studySummary.getId(),
+                ingestRequest.table("participant").path(gsPath(participantBlob)));
+            connectedOperations.ingestTableSuccess(studySummary.getId(),
+                ingestRequest.table("sample").path(gsPath(sampleBlob)));
+            connectedOperations.ingestTableSuccess(studySummary.getId(),
+                ingestRequest.table("file").path(gsPath(fileBlob)));
+
+            // Create a dataset!
+            MockHttpServletResponse datasetResponse =
+                connectedOperations.launchCreateDataset(studySummary, "ingest-test-dataset.json", "");
+            DatasetSummaryModel datasetSummary = connectedOperations.handleCreateDatasetSuccessCase(datasetResponse);
+            DatasetModel dataset = connectedOperations.getDataset(datasetSummary.getId());
+
+            Assert.assertThat(dataset.getTables().size(), is(equalTo(3)));
```

Comment:
This is still a pretty high-level check (ideally we'd inspect the actual rows in the dataset), but it's good enough for a syntax check for the new SQL. The test fails with a 500 ISE on the dataset creation step using the code on `develop`, and passes with my changes.

Reply:
Yeah, it would be nice to do a query or two against BQ to validate that the views show what you expect.
We don't yet have a way of exposing the BQ information through the REST API. You'd have to "cheat" and use the bigquery configuration. Perhaps add a TODO here?

Reply:
can you delete the dataset and study also?

Reply:
`connectedOperations` deletes them by default in its `teardown()` method, which we're calling in the `@After` method.